runtime/common/src/precompile/dex.rs

```diff
@@ -38,17 +38,17 @@ pub struct DexPrecompile<AccountId, AddressMapping, CurrencyIdMapping, Dex>(
 	PhantomData<(AccountId, AddressMapping, CurrencyIdMapping, Dex)>,
 );
 
-#[derive(Debug, Eq, PartialEq, TryFromPrimitive)]
-#[repr(u8)]
-enum Action {
-	GetLiquidityPool = 0,
-	GetLiquidityTokenAddress = 1,
-	GetSwapTargetAmount = 2,
-	GetSwapSupplyAmount = 3,
-	SwapWithExactSupply = 4,
-	SwapWithExactTarget = 5,
-	AddLiquidity = 6,
-	RemoveLiquidity = 7,
+#[derive(Debug, Eq, PartialEq, TryFromPrimitive, IntoPrimitive)]
+#[repr(u32)]
+pub enum Action {
+	GetLiquidityPool = 0xf4f31ede,
+	GetLiquidityTokenAddress = 0xffd73c4a,
+	GetSwapTargetAmount = 0x4d60beb1,
+	GetSwapSupplyAmount = 0xdbcd19a2,
+	SwapWithExactSupply = 0x579baa18,
+	SwapWithExactTarget = 0x9782ac81,
+	AddLiquidity = 0x4ea5efef,
+	RemoveLiquidity = 0xda613b51,
```

Comment:
IMO actions variant values should be generated automatically in the future improvement. Manual adding them could be error-prune.

Reply:
we have unit test to assert those value are correct so it is not too bad. but yes I think we can have some macro to generate them